src/locations/Configuration/index.tsx

```diff
@@ -0,0 +1,336 @@
+  mediaQuality: 'auto'
+};
+
+const codeBlockStyle = { fontSize: '12px', fontWeight: 'bold', border: '1px solid #ccc', padding: '2px 4px', borderRadius: '4px' };
+
+const ConfigScreen = () => {
+  const sdk = useSDK<ConfigAppSDK>();
+  const [parameters, setParameters] = useState<Parameters>(DEFAULT_PARAMETERS);
+  const [showAdvancedConfiguration, setShowAdvancedConfiguration] = useState(false);
+
+  const onConfigure = useCallback(async () => {
+    // This method will be called when a user clicks on "Install"
+    // or "Save" in the configuration screen.
+    // for more details see https://www.contentful.com/developers/docs/extensibility/ui-extensions/sdk-reference/#register-an-app-configuration-hook
+
+    // Get current the state of EditorInterface and other entities
+    // related to this app installation
+    const currentState = await sdk.app.getCurrentState();
+    console.log('currentState', currentState);
+    console.log('parameters', parameters);
+
+    return {
+      // Parameters to be persisted as the app configuration.
+      parameters: {
+        ...parameters,
+        installationUuid: parameters.installationUuid || window.crypto.randomUUID(),
+      },
+      // In case you don't want to submit any update to app
+      // locations, you can just pass the currentState as is
+      targetState: currentState,
+    };
+  }, [parameters, sdk]);
+
+  useEffect(() => {
+    // `onConfigure` allows to configure a callback to be
+    // invoked when a user attempts to install the app or update
+    // its configuration.
+    sdk.app.onConfigure(() => onConfigure());
+  }, [sdk, onConfigure]);
+
+  useEffect(() => {
+    (async () => {
+      // Get current parameters of the app.
+      // If the app is not installed yet, `parameters` will be `null`.
+      const currentParameters: Parameters | null = await sdk.app.getParameters<Parameters>();
+
+      if (currentParameters) {
+        setParameters({
+          ...DEFAULT_PARAMETERS,
+          ...currentParameters,
+        });
+      }
+
+      // Once preparation has finished, call `setReady` to hide
+      // the loading screen and present the app to a user.
+      sdk.app.setReady();
+    })();
+  }, [sdk]);
+
+  const mediaQualityOptions = [
+    { label: 'Auto', value: 'auto' },
+    ...Array.from({ length: 10 }, (_, i) => ({
+      label: `${(i + 1) * 10}`,
+      value: `${(i + 1) * 10}`,
+    })),
+  ];
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement | HTMLSelectElement>, value: string | boolean | number | null) => {
+    const key: keyof Parameters = event.target.name as keyof Parameters;
+    setParameters(prev => ({
+      ...prev,
+      [key]: value
+    }));
+  };
+
+  return (
+    <>
+      <GlobalStyles />
+      <div style={{
+        display: 'block',
+        position: 'absolute',
+        zIndex: -1,
+        top: 0,
+        width: '100%',
+        height: '300px',
+        backgroundColor: '#f4b21b',
+      }} />
```

Comment:
![image](https://github.com/user-attachments/assets/d94bd978-b66d-4774-a006-e2e101e8c9a2)

Is it possible for us to change the background color to match our brand?

Dropbox has done it
![image](https://github.com/user-attachments/assets/f5b7b427-f3b6-483c-9e5a-7e59fc209269)

Reply:
done